bib/cmd/bootc-image-builder/main.go

```diff
@@ -139,10 +140,13 @@ func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest
 	// to using containers storage in all code paths happened.
 	// We might want to change this behaviour in the future to match podman.
 	if !c.Local {
+		fmt.Printf("Pulling image %s (arch=%s)\n", c.Imgref, c.Architecture.String())
```

Comment:
Sorry, I just realized (after looking at the failing tests) this has the same issue as: https://github.com/osbuild/bootc-image-builder/pull/286#discussion_r1530421631 - when we use "--manifest" we expect a pure json output. 

So I guess we need to either use stderr (which is weird as it's not an error) or use logrus and in manifest mode set the level to error but in build mode to info (or something else I have not thought about :)